pkg/kubelet/kubelet_node_status.go

```diff
@@ -50,6 +50,12 @@ const (
 	// maxNamesPerImageInNodeStatus is max number of names per image stored in
 	// the node status.
 	maxNamesPerImageInNodeStatus = 5
+
+	//Label for storing provided IP addresses
+	LabelProvidedIPAddr = "beta.kubernetes.io/provided-node-ip"
```

Comment:
Yes, I'll move them to "k8s.io/apimachinery/pkg/apis/meta/v1", where rest of the constants reside. But incorporating that requires me to update apimachinery and then update godeps in kubernetes. 

I'll do that change after this PR goes through. Otherwise, that'll bottleneck this PR for another release.